varnish/README.md

```diff
@@ -14,60 +14,47 @@ This check collects varnish metrics regarding:
 It also submits service checks for the health of each backend.
 
 ## Setup
-
-Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying these instructions.
-
 ### Installation
 
 The Varnish check is included in the [Datadog Agent][3] package. No additional installation is needed on your server.
 
 ### Configuration
+#### Host
 
-1. Edit the `varnish.d/conf.yaml` file, in the `conf.d/` folder at the root of your [Agent's configuration directory][4] to start collecting your Varnish [metrics](#metric-collection) and [logs](#log-collection). See the [sample varnish.d/conf.yaml][5] for all available configuration options.
-
-2. [Restart the Agent][6].
+Follow the instructions below to configure this check for an Agent running on a host. For containerized environments, see the [Containerized](#containerized) section.
 
-#### Prepare Varnish
+##### Prepare Varnish
 
 If you're running Varnish 4.1+, add the `dd-agent` system user to the Varnish group using:
 
 ```
 sudo usermod -G varnish -a dd-agent
 ```
 
-#### Metric collection
-
-* Add this configuration block to your `varnish.d/conf.yaml` file to start gathering your [Varnish metrics](#metrics):
-
-  ```
-  init_config:
+##### Metric collection
 
-  instances:
-    - varnishstat: /usr/bin/varnishstat        # or wherever varnishstat lives
-      varnishadm: <PATH_TO_VARNISHADM_BIN>     # to submit service checks for the health of each backend
-  #   secretfile: <PATH_TO_VARNISH_SECRETFILE> # if you configured varnishadm and your secret file isn't /etc/varnish/secret
-  #   tags:
-  #     - instance:production
-  ```
+1. Edit the `varnish.d/conf.yaml` file, in the `conf.d/` folder at the root of your [Agent's configuration directory][4]. See the [sample varnish.d/conf.yaml][5] for all available configuration options.
 
-  If you don't set `varnishadm`, the Agent won't check backend health. If you do set it, the Agent needs privileges to execute the binary with root privileges. Add the following to your `/etc/sudoers` file:
-
-  ```
-  dd-agent ALL=(ALL) NOPASSWD:/usr/bin/varnishadm
-  ```
-
-  See the [sample varnish.yaml][5] for all available configuration options.
-
-* [Restart the Agent][6] to start sending Varnish metrics and service checks to Datadog.
+    ```
+      init_config:
```

Comment:
We should inline the new configuration file required format here: https://github.com/DataDog/integrations-core/blob/master/varnish/datadog_checks/varnish/data/conf.yaml.example#L1#L12

Reply:
Out of scope for this PR